packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -318,6 +342,38 @@ class Auth {
         )
     }
 
+    /**
+     * This is a wrapper method for ShopperCustomer API registerCustomer endpoint.
+     *
+     */
+    async register(
+        body: Argument<ShopperCustomers<ApiClientConfigParams>['registerCustomer']>['body']
+    ) {
+        const {
+            customer: {email},
+            password
+        } = body
+        const isGuest = false
+
+        // email is optional field from isomorphic library
+        // type CustomerRegistration
+        // here we had to guard it to avoid ts error
```

Comment:
Totally understand... I commented because, i'd imagine i'll get questions like "why do we do custom validations for the hook? shouldn't that be in the isormophic library?" 😛